src/app-layer-parser.c

```diff
@@ -1761,21 +1761,12 @@ void AppLayerParserRegisterProtocolParsers(void)
     SCRfbRegisterParser();
     SCMqttRegisterParser();
     SCRegisterPgsqlParser();
+    rs_pop3_register_parser();
```

Comment:
style: new code should use `SCPop3RegisterParser` style for public rust funcs like this